regtests/0317_hide_id/test.out

```diff
@@ -1,77 +1,77 @@
-=============== First
```

Comment:
Why is this file fully rewritten? Change of line ending?

Reply:
Yes, you can see dark green and dark red background where are the real modifications.